src/Halogen/Infinite/Scroll.purs

```diff
@@ -68,6 +71,7 @@ defaultFeedParams e =
   , pageSize: 10
   , hiddenPages: 3
   , preloadedPages: 3 
+  , enableTop: true
```

Comment:
New config option to enable loading/unloading from above. Disabling should mean that the library works reliably on every device since the scroll issues arise from scroll offset management when adding or removing from the top.